src/styles/catalog.scss

```diff
@@ -0,0 +1,13 @@
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  width: var(--card-area-width);
+  margin: 0 auto;
+  padding-top: 50px;
+  padding-left: 40px;
+  
+
+  & :nth-child(4n) {
```

Comment:
There's a space between the '&' and ':nth-child(4n)' selector which might cause unwanted results. In BEM methodology, '&' is used to denote a modifier of a block or an element. The space makes the selector look for ':nth-child(4n)' globally within the '.cards' class, not just on the immediate children. You should remove the space to correctly target every fourth direct child of the '.cards' class.